src/forms.js

```diff
@@ -32,11 +32,10 @@ fn.extend({
   },
 
   val(value) {
-    if (value === undefined) {
-      return this[0].value;
-    } else {
-      return this.each(v => v.value = value);
-    }
+    return ( value === undefined ?
+      this[0].value :
+      this.each(v => v.value = value)
+    );
   }
```

Comment:
to build on this... I know You, Ken and such are pretty hardcore JS developers and can probably read this kind of code like it's a children's book... but personally storing ternary structures in my brain is difficult... 

It also worries me that any regular joes like myself who use Cash but find bugs, or difficulties and want to contribute will be pretty put-off if htey can't debug the un-minified code themselves.

Perhaps if shaving bytes is a particularly large goal, we can investigate ways to compress the code even better with tools, rather than replacing all our expressions with ternaries ? :)